### DISCUSSION TITLE
Has Shaden-ui considered the idea of creating a Figma design pack for a comprehensive user interface kit

### DISCUSSION THREAD

**Discussion by Justin-Bento** [shadcn-ui/ui#1369] [Ideas]
I wanted to share that I've recently used your user-interface pack into one of my projects, and it has been a fantastic. The ease and it brought to my design process with Tailwind CSS was awesome.

I'm wondering, if there might be any discussions about introducing a Figma design pack in the future. As Figma's design capabilities and design-friendly environment could make it an excellent platform for such a resource.

Looking forward to your thoughts on this possibility!

**Comment by JamesSingleton** [comment]
Is this what you are looking for? https://ui.shadcn.com/docs/figma

**Reply by Justin-Bento** [reply]
Yes, it was... I feel a bit silly for asking, but thanks for sharing.

**Reply by JamesSingleton** [reply]
No problem!

**Comment by shakedlokits** [comment]
@JamesSingleton Is it still supported?
<img width="275" alt="image" src="https://github.com/shadcn-ui/ui/assets/2076274/edea0912-dbc6-4739-85f6-cb9cc2425b83">
The discrepancies and missing components causes quite a bit of miscommunications between design and engineering for us.

**Reply by Justin-Bento** [reply]
Thanks for the reply. Will be using this in an new project. :)
